runtime/pubsub/internal/noop/topic.go

```diff
@@ -0,0 +1,27 @@
+package noop
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/rs/zerolog"
+
+	"encore.dev/appruntime/exported/config"
+	"encore.dev/pubsub/internal/types"
+)
+
+type Topic struct{}
+
+var ErrNoop = errors.New(
+	"pubsub: this service is not configured to use this topic. " +
+		"Use pubsub.TopicRef in the service to get a reference and access to the topic from this service",
```

Comment:
If a service isn't configured to use the topic, how would it get a `TopicRef` in the first place? A `TopicRef` can't be passed over the wire. I think we should just drop the second sentence from the error.

Reply:
I thought the whole point of a TopicRef was to allow you to say "this service needs a reference to this topic"

Reply:
True, but I suppose the same could be said for all other infrastructure. "sqldb: this service is not configured to use this database. Use `sqldb.Named` in the service to get a reference and access to the database."